### PR TITLE
Change log level of timeout events in AbortOnTimeoutEventHandler

### PR DESCRIPTION
**Pull request by ecfm**
https://builds.cask.co/browse/CDAP-RUT1405

### CODE REVIEW

File: cdap-common/src/main/java/co/cask/cdap/common/twill/AbortOnTimeoutEventHandler.java [hunk 1]
```diff
@@ -78,14 +78,14 @@ public void initialize(EventHandlerContext context) {
...
-      LOG.info("Requested {} containers for runnable {}, only got {} after {} ms.",
+      LOG.warn("Requested {} containers for runnable {}, only got {} after {} ms.",
                event.getExpectedInstances(), event.getRunnableName(),
                event.getActualInstances(), System.currentTimeMillis() - event.getRequestTime());
 
       boolean pass = abortIfNotFull ? event.getActualInstances() == event.getExpectedInstances()
                                     : event.getActualInstances() != 0;
       if (!pass && (now - event.getRequestTime()) > abortTime) {
-        LOG.info("No containers for {}. Abort the application.", event.getRunnableName());
+        LOG.error("No containers for {}. Abort the application.", event.getRunnableName());
```
Comment (chtyim) [inline cdap-common/src/main/java/co/cask/cdap/common/twill/AbortOnTimeoutEventHandler.java]: Also log the application name and run id. Same for the log statement above.

Comment (ecfm) [reply cdap-common/src/main/java/co/cask/cdap/common/twill/AbortOnTimeoutEventHandler.java]: how do I get app name and run id? I don't see it in in `event`

Comment (chtyim) [reply cdap-common/src/main/java/co/cask/cdap/common/twill/AbortOnTimeoutEventHandler.java]: It is in the `EventHandlerContext`. You wrote that in Twill, remember?

**Comment by chtyim** [comment]
LGTM

**Pull request merged**